src/index.ts

```diff
@@ -81,7 +171,10 @@ export const handleTokenRequest = async (ctx: Context, request: Request) => {
 				'pkcs8',
 				privateKey,
 				{
-					name: ctx.isTest() ? 'RSA-PSS' : 'RSA-RAW',
+					// name: ctx.isTest() ? 'RSA-PSS' : 'RSA-RAW',
+					// RSA-RAW is handled directlry by blindrsa-tsst() 
+					// https://github.com/cloudflare/blindrsa-ts/blob/64dc207f079ca680d4db6cd5a22f5bce6f1b28e8/src/blindrsa.ts#L168
+					name: 'RSA-PSS',
```

Comment:
this should not change
you should consider adding the importKey mock for the test, as is done on [cloudflare/privacypass-ts](https://github.com/cloudflare/privacypass-ts/blob/main/test/jest.setup-file.ts#L45-L91)